src/som/vmobjects/SClass.java

```diff
@@ -341,4 +360,12 @@ public String toString() {
   public MaterializedFrame getContext() {
     return context;
   }
+
+  public AbstractSerializationNode getSerializer() {
+    return serializer;
+  }
+
+  public void setSerializer(final AbstractSerializationNode serializer) {
```

Comment:
Is this setter strictly necessary? Can't this be done via the constructor?
Then the field could be final, too.